### PR TITLE
Update Yaml labels

### LINKED ISSUE #898: Update Yaml labels for new Testground Repo
Updating labels to include:

Kind/investigation
Kind/chore
Status/ Approved-Waiting
Status/Changes Requested
Status/Waiting for Review

### PR DESCRIPTION
**Pull request by Robmat05**
Labels to add to YAML

Kind/investigation
Kind/chore
Status/ Approved-Waiting
Status/Changes Requested
Status/Waiting for Review

---

Closes #898.

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -63,6 +63,12 @@
 - name: kind/problem
   color: c92712
   description: "Kind: Problem"
+- name: kind/Investigation
```
Comment (raulk) [inline .github/labels.yml:66]: 1. How is this different to a spike?
2. Please follow capitalisation for label names: small case, with hyphens for spaces.

File: .github/labels.yml [hunk 2]
```diff
@@ -229,6 +235,15 @@
 - name: status/done
   color: edb3a6
   description: "Status: Done"
+- name: status/Approved-Waiting
+  color: edb3a6
+  description: "Status: "Approved-Waiting for Merge"
```
Comment (raulk) [inline .github/labels.yml:240]: there are three quotes here.

File: .github/labels.yml [hunk 3]
```diff
@@ -229,6 +235,15 @@
 - name: status/done
   color: edb3a6
   description: "Status: Done"
+- name: status/Approved-Waiting
```
Comment (raulk) [inline .github/labels.yml:238]: Please follow capitalisation for label names: small case, with hyphens for spaces: `status/approved-waiting`.

File: .github/labels.yml (hunk 2 above)
Comment (raulk) [reply .github/labels.yml:240]: suggested: "Status: Approved and awaiting merge".

File: .github/labels.yml [hunk 4]
```diff
@@ -229,6 +235,15 @@
...
+  color: edb3a6
+  description: "Status: "Approved-Waiting for Merge"
+- name: status/Changes Requested
+  color: edb3a6
+  description: "Status: "Changes Requested in Review"
```
Comment (raulk) [inline .github/labels.yml:243]: same as above

File: .github/labels.yml [hunk 5]
```diff
@@ -229,6 +235,15 @@
 - name: status/done
   color: edb3a6
   description: "Status: Done"
+- name: status/Approved-Waiting
+  color: edb3a6
+  description: "Status: "Approved-Waiting for Merge"
+- name: status/Changes Requested
```
Comment (raulk) [inline .github/labels.yml:241]: same as above.

File: .github/labels.yml (hunk 1 above)
Comment (Robmat05) [reply .github/labels.yml:66]: I can see that a spike and investigation are similar. To my way of thinking a spike is a surgically specific look into an problem or question. eg Look into whether InfluxDB or ElastiSearch works better with Docker Runner.

An investigation would be more general and a bit more time consuming. eg…  Look for the best open-source time series database

**Pull request merged**